_content/v2-preview/java/_api.md

```diff
@@ -105,6 +121,7 @@ client.onMessage("my-uno", "led", new BiConsumer<String, Object>() {
         System.out.println("Message: " + devId + " " + data);
 });
 ```
+  > When a field is specified, `cb.data` holds the `String` representation of the field value.
```

Comment:
Doesn't need to be a string. It can also be a boolean or array or object. Basically a selection of `payload_fields`